relayer-engine/src/config/index.ts

```diff
@@ -37,15 +37,22 @@ export enum Mode {
 }
 export type NodeURI = string;
 
+export interface RedisConfig {
+  host: string;
+  port: number;
+  username?: string;
+  password?: string;
+  tls?: boolean;
+}
+
 export interface CommonEnv {
   namespace?: string;
   logLevel?: string;
   promPort?: number;
   readinessPort?: number;
   logDir?: string;
   storeType: StoreType;
```

Comment:
Consider removing this if the redis config block is now required

Reply:
I'll make redis optional, thanks for the catch.